### ISSUE TITLE
Migration renaming in 0.18.1 is causing a Django migration error in django migrate command

### ISSUE THREAD

**Issue by ignaciocabeza** [peopledoc/procrastinate#382]
Hi everybody,

I recently updated procrastinate 0.16.0 to 0.18.1 and the command `python manage.py migrate` is failing with the following error:
```
Traceback (most recent call last):
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 330, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 371, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 85, in wrapped
    res = handle_func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/commands/migrate.py", line 95, in handle
    executor.loader.check_consistent_history(connection)
  File "/usr/local/lib/python3.8/site-packages/django/db/migrations/loader.py", line 302, in check_consistent_history
    raise InconsistentMigrationHistory(
django.db.migrations.exceptions.InconsistentMigrationHistory: Migration procrastinate.0002_drop_started_at_column is applied before its dependency procrastinate.0001_initial on database 'default'.
```

The issue was generated because this commit: https://github.com/peopledoc/procrastinate/commit/1e989e00cd20e9e866c4c6f66b62ca2251aba1bf where the `0001_baseline` migration file changed to `0001_initial`: `0  procrastinate/sql/migrations/{baseline-0.5.0.sql → 00.00.00_01_initial.sql}`. Django doesn't know that is the same file and is creating a  not applied migration for `0001`:

![image](https://user-images.githubusercontent.com/479689/105126943-c35f1d80-5b1a-11eb-97b3-026ed04fd25b.png)

As a reference, In version 0.16.0 we had:
![Screen Shot 2021-01-20 at 12 14 28 pm](https://user-images.githubusercontent.com/479689/105126880-a1659b00-5b1a-11eb-999d-331a8bdd3aa2.png)

Is there a reason for changing the name of the migration? Is is possible to rollback to the original name `0001_baseline`?

**Comment by frinzekt** [comment]
I am encountering the same issue !!

**Comment by ewjoachim** [comment]
Ah, yes, we reworked the migrations names so that the alphanumerical sort on the same would produce the migrations in correct order, and since we had stopped using pum, which was the reason the initial migration was named "baseline" at first, we took the opportunity to give it a proper name. I didn't anticipate there would be a problem with Django.

Would it be a problem for you to rename the migration in your DB instead ?

```python
In [1]: from django.db.migrations.recorder import MigrationRecorder

In [2]: MigrationRecorder.Migration.objects.filter(app="procrastinate", name="0001_baseline").update(name="0001_initial")
Out[2]: 1
``` 

I've reproduced the bug, and locally, running the ORM command from above on a database migrated with 0.16.0 was enough to let me use migrations from 0.17.0 and later.
